File: monitor/bin/rb/helpers/db_production.rb
require 'pg'

# Sets up a DB connection

puts 'connecting to:'
puts "host: #{ENV['POSTGRES_HOST_PRODUCTION']}"
puts "dbname: #{ENV['POSTGRES_DB_NAME_PRODUCTION']}"

puts 'ok? (y/n)'
ok = gets.chomp == 'y'
if !ok then exit 1 end

def db
  return PG.connect(
    host: ENV['POSTGRES_HOST_PRODUCTION'],
    dbname: ENV['POSTGRES_DB_NAME_PRODUCTION'],
    user: ENV['POSTGRES_USER_PRODUCTION'],
    password: ENV['POSTGRES_PASSWORD_PRODUCTION']
  )
end

File: monitor/bin/js/db-production.js
const { Client } = require('pg');

const client = new Client({
  host: process.env.POSTGRES_HOST_PRODUCTION,
  port: process.env.POSTGRES_PORT_PRODUCTION,
  database: process.env.POSTGRES_DB_NAME_PRODUCTION,
  user: process.env.POSTGRES_USER_PRODUCTION,
  password: process.env.POSTGRES_PASSWORD_PRODUCTION,
});

client.connect();

module.exports = client;
